docs/source/index.rst

```diff
@@ -0,0 +1,291 @@
+
+            To The Quickstart Page
+
+    .. grid-item-card::
+        :img-top: _images/index/lightbulb.png
+
+        Examples
+        ^^^^^^^^
+        Have some basic experience with ``cluster_generator``, but want to see a guide on how to execute a particular task? Need
+        to find some code to copy and paste? The examples page contains a wide variety of use case examples and explainations
+        for all of the various parts of the ``cluster_generator`` library.
+
+        +++
+
+        .. button-ref:: examples
+            :expand:
+            :color: secondary
+            :click-parent:
+
+            To the Examples Page
+
+    .. grid-item-card::
+        :img-top: _images/index/book.svg
+
+        User References
+        ^^^^^^^^^^^^^^^^
+        The user guide contains comprehensive, text based explainations of the backbone components of the ``cluster_generator`` library.
+        If you're looking for information on the underlying code or for more details on particular aspects of the API, this is your best resource.
+
+        +++
+
+        .. button-ref:: codes
+            :expand:
+            :color: secondary
+            :click-parent:
+
+            To the User Guide
+
+    .. grid-item-card::
+        :img-top: _images/index/api_icon.png
+
+        API Reference
+        ^^^^^^^^^^^^^
+
+        Doing a deep dive into our code? Looking to contribute to development? The API reference is a comprehensive resource
+        complete with source code and type hinting so that you can find every detail you might need.
+
+        +++
+
+        .. button-ref:: api
+            :expand:
+            :color: secondary
+            :click-parent:
+
+            API Reference
+
+
+Contents
+========
+.. raw:: html
+
+   <hr style="height:10px;background-color:black">
+
+.. toctree::
+   :maxdepth: 1
+
+   Getting_Started
+   gravity
+   models
+   collections
+   examples
+   api
+
+Related Projects
+================
+
+.. grid:: 2
+    :padding: 3
+    :gutter: 5
+
+    .. grid-item-card::
+        :img-top: _images/index/PyXSIM.png
+
+        PyXSIM
+        ^^^^^^
+
+        Convert your ``cluster_generator`` systems into synthetic photon event lists for use in simulating observations using X-ray observatories using PyXSIM.
+        ``cluster_generator`` is designed to easily interface with this library to provide as much ease as possible when building simulated observations of clusters.
+
+        +++
+
+        .. button-link:: http://hea-www.cfa.harvard.edu/~jzuhone/pyxsim/
+            :expand:
+            :color: secondary
+            :click-parent:
+
+            Documentation
+
+
+    .. grid-item-card::
+        :img-top: _images/index/SOXS.png
+
+        SOXS
+        ^^^^
+
+        Coupled with PyXSIM, SOXS is a instrument simulation tool for turning mock photon counts into realistic X-ray observations specific to the
+        behavior of specific instruments like CHANDRA, XMM-Newton, and NuSTAR.
```

Comment:
I would write it as *Chandra*, *XMM-Newton*, and *NuSTAR*. Though only the first is actually supported directly in SOXS.

Reply:
@jzuhone Fixed in most recent commit.